docker/docker-compose.yml

```diff
@@ -157,3 +185,22 @@ services:
         hard: -1
     ports:
       - '9200'
+    labels:
```

Comment:
NOTE: @manekenpix recently changed this to expose 9200 in development, since we can't do fancy domain names for localhost.  Check what's on `master`.

Reply:
@humphd @manekenpix I don't see the Labels part on master - should it be removed?

Reply:
Follow the way things are on `master` yes. NOTE: we define things in different places now: the docker-compose.yml is our base file, and development.yml are overrides/extras for development/CI; production.yml for production.  So look in all three places.